packages/edit-site/src/components/block-editor/index.js

```diff
@@ -138,8 +148,14 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 			__unstableFetchMedia: fetchMedia,
 			__experimentalBlockPatterns: blockPatterns,
 			__experimentalBlockPatternCategories: blockPatternCategories,
+			__experimentalFetchPageEntities: fetchPagesEntities,
 		};
-	}, [ storedSettings, blockPatterns, blockPatternCategories ] );
+	}, [
+		storedSettings,
+		blockPatterns,
+		blockPatternCategories,
+		fetchPagesEntities,
```

Comment:
Do you think there's a chance this might cause a perf problem because `fetchPagesEntities` is always a new function reference?

I'm hoping the fact that `useSelect` has `[]` as deps means it will only run once and thus `fetchPagesEntities` will remain a consistent reference but if that's not true then it will cause problems as the `useMemo` will be invalidated for the entire editor settings.